README.md

```diff
@@ -53,9 +53,17 @@ php artisan budget:update
 
 ## Docker
 
-You can run Budget using Docker. Spinning it up (`docker-compose up`) will create 3 containers–for MySQL, PHP and NGINX.
+You can run Budget using Docker Compose. Docker Compose starts 2 containers - one for the database and the other for the application itself.
 
-*Disclaimer–currently the Docker setup is missing cronjobs and the queue worker.*
+1. Clone the repository
+2. Copy the `.env.example` to `.env`, make sure to review/replace any other value as you wish
```

Comment:
Maybe we should detail exactly what configuration options are needed for the project to run at a minimal.